### PR TITLE
CH-22 Refactor getGamesInWeek

### PR DESCRIPTION
**Pull request by Jpena91**
Hi

### CODE REVIEW

File: __tests__/unit/FunctionTest.unitTest.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ describe('../util/nbaUtils', () => {
     console.log('getTeamId():\nExpected: ' + expectedId + '\nReceived: ' + teamId)
     expect(teamId).toBe(expectedId)
   })
-
+/*old and archived again because I was told to
```
Comment (shibachurro) [inline __tests__/unit/FunctionTest.unitTest.js:16]: lol

**Comment by shibachurro** [comment]
looks good to me man @Laturine any concerns?

File: __tests__/unit/FunctionTest.unitTest.js (hunk 1 above)
Comment (Laturine) [reply __tests__/unit/FunctionTest.unitTest.js:16]: Let's delete the old/archived test.

File: util/nbaUtils.js [hunk 2]
```diff
@@ -70,6 +70,7 @@ const getDaysInWeek = function (monDate) {
  * @return {number} games played for given team in given week
  */
 
+/*old and archived because I was told to
```
Comment (Laturine) [inline util/nbaUtils.js:73]: Let's delete this old and archived test as well

File: app.js [hunk 3]
```diff
@@ -13,8 +13,8 @@ client.on('message', async message => {
     message.channel.send('hellow world.')
   }
 
-  if (message.content === 'days') {
-    message.channel.send(nbaUtil.getDaysOfWeek(nbaUtil.getCurrentMonday()))
+  if (message.content === 'days') {//just changed 'Of' to 'In'...it bothered me ok?
```
Comment (Laturine) [inline app.js:16]: You can delete this comment as well.

File: util/nbaUtils.js [hunk 4]
```diff
@@ -115,6 +116,18 @@ const getDaysPlayedOn = function (data, lastPlayedIndex, daysInWeek) {
 // const getGamesInWeekTest = function (data, lastPlayedIndex, daysInWeek) {
 //
 // }
+//getGamesInWeekTest
+const getGamesInWeek = function (data, lastPlayedIndex, daysInWeek) {
+  let gamesPlayed = 0
+  data.slice(lastPlayedIndex).forEach(game => {
+    // logic to search all games
+    let gameDate = game.startDateEastern
```
Comment (Laturine) [inline util/nbaUtils.js:124]: We're not changing gameDate correct? this can be const?

File: __tests__/unit/FunctionTest.unitTest.js [hunk 5]
```diff
@@ -29,6 +29,24 @@ describe('../util/nbaUtils', () => {
     console.log('getGamesInWeek: \nExpected: ' + expected + '\nReceived: ' + received)
     expect(received).toBe(expected)
   })
+*/
+//new "test"
+test('getGamesInWeek() returns number of games during a given week', async () => {
+  let teamId = '1610612737'
```
Comment (Laturine) [inline __tests__/unit/FunctionTest.unitTest.js:35]: Let's change this to const.

Comment (Laturine) [reply __tests__/unit/FunctionTest.unitTest.js:35]: and any other variable that doesn't get changed.

**Comment by Jpena91** [comment]
This is invalid now though right???

**Comment by shibachurro** [comment]
closing to re-push updated branch

**Pull request closed without merging**